lib/rules/lines-around-comment.js

```diff
@@ -93,6 +93,12 @@ module.exports = {
                     },
                     allowArrayEnd: {
                         type: "boolean"
+                    },
+                    ignorePattern: {
+                        type: "string"
+                    },
+                    applyDefaultPatterns: {
```

Comment:
I also think the option name isn't clear. Thoughts on updating it to `applyDefaultIgnorePatterns` in both places (making it backwards compatible where we need to)?